RunTests.ps1

```diff
@@ -0,0 +1,77 @@
+
+$DnxRuntimes = @(
+	'dnx-clr-win-x86.1.0.0-beta4', 
+	'dnx-coreclr-win-x86.1.0.0-beta4');
```

Comment:
global.json - I don't see anything specified there. We can enumerate all the runtimes which are in the .dnx/runtime folder and run against them.